Automattic/wp-calypso
# Post-purchase Upsell: Update CSS and check for 7 or 14 money back guarantee

Related to https://github.com/Automattic/dotcom-forge/issues/1596

## Proposed Changes

* Update the style to match the design [here](https://github.com/Automattic/dotcom-forge/issues/1596#issuecomment-1425597469).
* Update so the copy specifies a 7-day money back guarantee for monthly plans, and 14-day for annual plans.

Before | After
--|--
![new-post-purchase-before](https://user-images.githubusercontent.com/140841/218210926-d80e075c-2a69-492f-bc1c-865674e80370.png)  |  ![new-post-purchase-after](https://user-images.githubusercontent.com/140841/218210973-70015318-3474-4e1d-b85b-41c260fb5bf2.png)

## Testing Instructions



* Load this PR
* Using a free site, purchase an annual Premium plan. After checking out, you'll see this upsell page.
* How does it look? Does it match the design [here](https://github.com/Automattic/dotcom-forge/issues/1596#issuecomment-1425597469)?
* Does it look ok in different browsers, resolutions, and on mobile?
* Does it say "14-day money back guarantee"?
* Test again purchasing a monthly Premium plan. Again, you'll see this upsell.
* It should say "7-day money back guarantee" for monthly plans.

## Pre-merge Checklist
